pygmt/helpers/utils.py

```diff
@@ -550,3 +551,37 @@ def args_in_kwargs(args, kwargs):
     return any(
         kwargs.get(arg) is not None and kwargs.get(arg) is not False for arg in args
     )
+
+
+def return_table(session, output_type, vfile, colnames):
+    """
+    Return an output table from a virtual file based on the output type.
+
+    Parameters
+    ----------
+    session : :class:`pygmt.clib.Session`
+        The current session.
+    output_type : str
+        The output type. Can be ``"pandas"``, ``"numpy"``, or ``"file"``.
+    vfile : str
+        The virtual file name.
+    colnames : list of str
+        The column names for the :class:`pandas.DataFrame` output.
```

Comment:
Not sure if `colnames` is a good name. In the module wrappers, we're inconsistently using `column_names` or `dataframe_header`.

Reply:
OK, I've updated it to `column_names` in 3d5147d422763ef24b3a9b415933e00d535f8e92.